spring-xd-shell/src/test/java/org/springframework/xd/shell/command/ModuleListTests.java

```diff
@@ -16,18 +16,16 @@
 
 package org.springframework.xd.shell.command;
 
-import static com.google.common.collect.Lists.newArrayList;
-import static com.google.common.collect.Maps.transformValues;
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasEntry;
-import static org.hamcrest.Matchers.hasSize;
-import static org.junit.Assert.assertThat;
+import static com.google.common.collect.Lists.*;
```

Comment:
Seems like your rules for imports are off (or is this what we should be using?)

Also noticed in another class above, but this has no other change apart from imports